### ISSUE TITLE
onDblClick is being transformed to onDoubleClick, which is invalid

### ISSUE THREAD

**Issue by goenning** [solidjs-community/eslint-plugin-solid#46]
**Describe the bug**
Whenever I try to use the onDblClick event on a button tag, the eslint-plugin transforms to onDoubleClick, which is in invalid event on a button.

**To Reproduce**
My config:

```js
module.exports = {
  env: {
    browser: true,
    es2021: true,
  },
  extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended", "plugin:solid/recommended", "prettier"],
  parser: "@typescript-eslint/parser",
  parserOptions: {
    ecmaVersion: "latest",
    sourceType: "module",
  },
  plugins: ["@typescript-eslint"],
  rules: {
    "@typescript-eslint/consistent-type-imports": "error",
  },
};
```

```html
<button onDblClick={() => console.log("Hey")}>
Double Click me
</button>
```

**Expected behavior**

The onDblClick event should not be transformed, it's a valid name.

**Screenshots**
<img width="984" alt="Screenshot 2022-11-27 at 06 34 58" src="https://user-images.githubusercontent.com/94755/204122620-1ac0c3d5-603d-4c4a-a5d9-a39448aa8de6.png">

**Environment (please complete the following information):**

- OS: macOS 13
- Node version (`node --version`): 16.17.0
- `eslint-plugin-solid` version (`npm list eslint-plugin-solid`/`yarn why eslint-plugin-solid`): eslint-plugin-solid@0.8.0
- `eslint` version (`npm list eslint`/`yarn why eslint`): eslint@8.28.0

**Additional context**

- [ ] I would be willing to contribute a PR to fix this issue

**Comment by joshwilsonvu** [comment]
Good catch—thanks so much for opening the issue! Fixed in `v0.9.1`.
